libs/measurement-history/src/lib/measurement-history.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+
+import MeasurementHistory from './measurement-history';
+
+describe('MeasurementHistory', () => {
+  it('should render successfully', () => {
+    //const { baseElement } = render(<MeasurementHistory />);
+    //expect(baseElement).toBeTruthy();
```

Comment:
Could be removed?

Reply:
No, because I want to do it later, it just failed because the get request requires to be wrapper in the Auth Provider. But as I commented, I will do the tests on the weekend